<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Debugger X64 (OllyDbg Clone)</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionMemoryMap"/>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionRun"/>
    <addaction name="actionPause"/>
    <addaction name="actionRestart"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionStepInto"/>
    <addaction name="actionStepOver"/>
    <addaction name="actionRtr"/>
    <addaction name="separator"/>
    <addaction name="actionCommand"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Plugins</string>
    </property>
    <addaction name="actionScylla"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuDebug"/>
   <addaction name="menuPlugins"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionRestart"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionRun"/>
   <addaction name="actionPause"/>
   <addaction name="separator"/>
   <addaction name="actionStepInto"/>
   <addaction name="actionStepOver"/>
   <addaction name="separator"/>
   <addaction name="actionRtr"/>
   <addaction name="separator"/>
   <addaction name="actionGoto"/>
   <addaction name="separator"/>
   <addaction name="actionLog"/>
   <addaction name="actionMemoryMap"/>
   <addaction name="separator"/>
   <addaction name="actionScylla"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="cmdBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/folder-horizontal-open.png</normaloff>:/icons/images/folder-horizontal-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/control-exit.png</normaloff>:/icons/images/control-exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+X</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/arrow-run.png</normaloff>:/icons/images/arrow-run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/control-pause.png</normaloff>:/icons/images/control-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/arrow-restart.png</normaloff>:/icons/images/arrow-restart.png</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F2</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/control-stop.png</normaloff>:/icons/images/control-stop.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Alt+F2</string>
   </property>
  </action>
  <action name="actionStepInto">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/arrow-step-into.png</normaloff>:/icons/images/arrow-step-into.png</iconset>
   </property>
   <property name="text">
    <string>Step into</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionStepOver">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/arrow-step-over.png</normaloff>:/icons/images/arrow-step-over.png</iconset>
   </property>
   <property name="text">
    <string>Step over</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionCommand">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/terminal-command.png</normaloff>:/icons/images/terminal-command.png</iconset>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Return</string>
   </property>
  </action>
  <action name="actionRtr">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/arrow-step-rtr.png</normaloff>:/icons/images/arrow-step-rtr.png</iconset>
   </property>
   <property name="text">
    <string>Execute till return</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F9</string>
   </property>
  </action>
  <action name="actionMemoryMap">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/memory-map.png</normaloff>:/icons/images/memory-map.png</iconset>
   </property>
   <property name="text">
    <string>Memory Map</string>
   </property>
   <property name="shortcut">
    <string>Alt+M</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/alphabet/L.png</normaloff>:/icons/images/alphabet/L.png</iconset>
   </property>
   <property name="text">
    <string>Log Window</string>
   </property>
   <property name="shortcut">
    <string>Alt+L</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/alphabet/A.png</normaloff>:/icons/images/alphabet/A.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionGoto">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/geolocation-goto.png</normaloff>:/icons/images/geolocation-goto.png</iconset>
   </property>
   <property name="text">
    <string>Goto</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionScylla">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/icons/images/scylla.png</normaloff>:/icons/images/scylla.png</iconset>
   </property>
   <property name="text">
    <string>Scylla</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
